Product Designer/assets/css/styles.css

```diff
@@ -0,0 +1,203 @@
+    display: flex;
+    justify-content: end;
+    font-size: 2.5em;
+    font-weight: 900;
+    height: 163px;
+    align-items: center;
+}
+.mailSection h6{
+    letter-spacing: 0.5px;
+    font-size: 0.6em;
+}
+
+/* NAME SECTION STYLES */
+.nameSection{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 485px;
+    width: 74%;
+    margin: 0 auto;
+}
+.name{
+    font-size: 150px;
+    margin: 0;
+}
+.name span{
+    font-weight: 300;
+    letter-spacing: 3px;
+}
+.role span{
+    color: #4b90e2;
+}
+.role{
+    font-size: 47px;
+}
+.mobileEmail{
+    display: none;
+}
+
+/* PRODUCTS SECTION  */
+.productsContainer{
+    display: flex;
+    flex-wrap: wrap;
+    width: 80%;
+    margin: 0 auto;
+    justify-content: space-between;
+    row-gap: 50px;
+}
+.productsContainer img{
+    width: 427px;
+}
+
+
+/* LOGO SECTION STYLES */
+.logoContainer{
+    width: 80%;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+}
+.logoTitle{
+    margin-top: 120px;
+    margin-bottom: 120px;
+    font-size: 47px;
+    color: #CECECE;
+    font-weight: 700;
+}
+.logoContainer span{
+    color: #B8E985;
+}
+.logoRow{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;
+}
+.logoContainer img{
+    width: 120px;
+}
+#adidas{
+    width: 100px;
+}
+
+/* FOOTER SECTION STYLES */
+.footer{
+    display: flex;
+    margin: 0 auto;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 600px;
+    width: 80%;
+    margin-top: 100px;
+}
+.iconsSection{
+    display: flex;
+    width: 400px;
+    justify-content: space-around;
+    align-items: center;
+    height: 240px;
+}
+.iconsSection img{
+    height: 80px;
+}
+.breakLines{
+    height: 27px;
+    width: 75%;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+}
+.shortLine{
+    background-color: white;
+    height: 2px;
+    width: 45px;
+    margin-right: 30px;
+}
+.longLine{
+    background-color: white;
+    height: 2px;
+    width: 210px;
+}
+
+/* STYLES FOR SMALLER SCREENS */
+@media only screen and (max-width: 1163px){
+    .mailSection{
+        color: black;
+        margin-top: 200px;
+        visibility: hidden;
+    }
+    .mobileEmail{
+        display: inline-block;
+        margin: 0px;
+        margin-bottom: 200px;
+        font-size: 32px;
+        font-weight: 900;
+        text-align: center;
+    }
+    .name{
+        font-size: 120px;
+    }
+    .role{
+        font-size: 65px;
+        margin: 0;
+    }
+    .productsContainer{
+        justify-content: center;
+    }
+    .productsContainer img{
+        width: 100%;
+    }
+    .logoContainer img{
+        padding: 80px;
+        width: 50%;
+    }
+    #adidas{
+        width: 50%;
+        padding: 80px;
+    }
+    .logoTitle{
+        font-size: 3vh;
```

Comment:
Why use vh here ?

Reply:
It was unnessary code that i have used for cross checking now its all removed